### DISCUSSION TITLE
Filter Data with Relationship

### DISCUSSION THREAD

**Discussion by betterappsllc** [Budibase/budibase#2871] [Help] [answered]
I have two tables that I have created.

1. EMS
2. PDF

There can be many PDFs for one EMS.

In the actual screen design I'm just trying to select an EMS and have it display the PDFs (filtered based on the selected EMS) in a table below.  I tried using a Relationship P
<img width="1440" alt="Screen Shot 2021-10-03 at 7 33 08 PM" src="https://user-images.githubusercontent.com/63764943/135775694-fa41a244-9b89-4234-8f1d-98cdb5ec0f3f.png">
icker in a form to select an EMS from the PDFs, but I couldn't figure out how to filter the data.  When I add an additional data provider to get the filtered data I don't see any columns for EMS in the filter area.  I do see the relationship picker data though, so I feel like I am close.  I'm probably just going about this the wrong way, but any help would be appreciated!

**Comment by mjashanks** [comment, marked as answer]
Hi There. This is currently a limitation of Budibase. We intend to address this in the next few months - you can track the issue here: https://github.com/Budibase/budibase/issues/2316.

**Reply by betterappsllc** [reply]
Thanks for the reply! I can most likely work around this for now. Really enjoying the software by the way.

**Reply by pepeday** [reply]
Hey there. What's the work around when using internal bb dB?
When working with external dB I think it's easier since the relational fields are exposed but there's lots of work to be done to make it work correctly.

**Reply by mjashanks** [reply]
There is no workaround at the moment :/

**Comment by xnohat** [comment]
Hi there,
currently version 1.0.46 does have any workaround for filter relationship in SQL tables or Internal database tables ?
